custom/icds_reports/tasks.py

```diff
@@ -1209,6 +1214,40 @@ def build_incentive_report(agg_date=None):
                 build_incentive_files.delay(location, agg_date, file_format, 3, location.parent.parent, location.parent)
 
 
+def is_launched_awc_incentive_report(launched_awc_ids):
+    awc_ids = launched_awc_ids.values_list('awc_id', flat=True)
+    awc_launched_count_from_aggregate = AggAwc.objects.filter(aggregation_level=5, awc_id__in=awc_ids).all()
+    csv_columns = ['awc_id', 'AwwIncentiveReport', 'AggAwc']
+    csv_data = []
+    for awc in launched_awc_ids:
+        awc_from_aggregate = awc_launched_count_from_aggregate.filter(awc_id=awc[0]).values_list('awc_id',
+                                                                                                 'is_launched')[0]
+        is_launched_from_awc = (lambda: False, lambda: True)[awc_from_aggregate[1].lower() == 'yes']()
```

Comment:
This line is difficult to parse, and appears to have the same functionality as just the inner part `awc_from_aggregate[1].lower() == 'yes'`. It would be much easier to read if it was reduced to that.